packages/orchestration/test/ibc-mocks.test.ts

```diff
@@ -74,3 +75,21 @@ test('build Query Packet', t => {
     'eyJkYXRhIjoiQ2pvS0ZBb0xZMjl6Ylc5ek1YUmxjM1FTQlhWaGRHOXRFaUl2WTI5emJXOXpMbUpoYm1zdWRqRmlaWFJoTVM1UmRXVnllUzlDWVd4aGJtTmwiLCJtZW1vIjoiIn0=',
   );
 });
+
+test('build QueryDelegatorDelegationsResponse', t => {
```

Comment:
I think I ran into trouble parsing before. Do you know whether we already have a test for that? It seems pretty cheap to add one here.